src/main/routes.ts

```diff
@@ -155,6 +166,11 @@ export class Routes {
     app.get(C100_RETRIVE_CASE, errorHandler(new GetCaseController().getC100ApplicantCase));
     app.get(C100_DOWNLOAD_APPLICATION, errorHandler(new ApplicationDownloadController().download));
 
+    const tsDraftController = new TSDraftController();
```

Comment:
Instead you can export an instance of the TSDraftController from the class file itself and use it here.